gp-templates/translation-row.php

```diff
@@ -12,7 +16,7 @@
 $can_reject_self = ( isset( $t->user->user_login ) && $user->user_login === $t->user->user_login && 'waiting' === $t->translation_status );
 ?>
 
-<tr class="preview <?php echo $status_class.' '.$warning_class.' '.$priority_class ?>" id="preview-<?php echo $t->row_id ?>" row="<?php echo $t->row_id; ?>">
+<tr class="preview <?php esc_attr( gp_translation_row_classes( $t ) ); ?>" id="preview-<?php echo esc_attr( $t->row_id ) ?>" row="<?php echo esc_attr( $t->row_id ); ?>">
```

Comment:
`gp_translation_row_classes()` echos the classes, `esc_attr()` should be removed.

To make the sniffer happy this function can be added to the list of auto-escaped functions: https://github.com/GlotPress/GlotPress-WP/blob/develop/phpcs.ruleset.xml#L16